src/style.css

```diff
@@ -1 +1,61 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
```

Comment:
You should add a fallback font to the `font-family` property in case `Avenir` font doesn't load. For example, you can use a generic font family like `sans-serif` as a fallback.